shared/bundle_analysis/parser.py

```diff
@@ -183,6 +200,8 @@ def _parse_assets_event(self, prefix: str, event: str, value: str):
                     name=self.asset.name,
                     normalized_name=self.asset.normalized_name,
                     size=self.asset.size,
+                    uuid=str(uuid.uuid4()),
```

Comment:
You are right, this is just the part 1 of the change to make linking assets across different commits work. By default we will create the `uuid` for an asset for this PR, for the next PR as the next step of the parsing process it will look at the previous commit's asset, if the algorithm thinks its the same then it will update the current asset's `uuid` to be the same as the previous, otherwise it will remain the same.

I choose to initialize an `uuid` on creation and then update it in the following parsing step is because the parsing code is already kind of convoluted, so setting the final `uuid` right now it would be a nightmare.